data/src/main/java/com/ionutconstantinnicolae/data/feature/news/local/ArticleEntity.java

```diff
@@ -0,0 +1,22 @@
+package com.ionutconstantinnicolae.data.feature.news.local;
+
+import androidx.annotation.Nullable;
+import androidx.room.Entity;
+import androidx.room.PrimaryKey;
+
+
+public class ArticleEntity {
+
+    @PrimaryKey(autoGenerate = true)
+    @Nullable
+    public Integer id;
+
+    public String imageUrl;
+
+    public String title;
+
+    public String content;
+
+    public String description;
```

Comment:
let's make these fields final (besides the id) with a nice constructor, immutable classes are less prone to bugs due to the limiting of the app states